src/quo/components/share/share_qr_code/schema.cljs

```diff
@@ -0,0 +1,74 @@
+(ns quo.components.share.share-qr-code.schema
```

Comment:
@ilmotta could you have a look at this as well please?

Reply:
Hey @clauxx.

The schemas look good. Good usage of multi schemas too. Also the usage of a separate `schema` namespace for larger schemas is a nice touch.

Reply:
@ilmotta Thanks!